lib/bower-rails/performer.rb

```diff
@@ -197,5 +199,10 @@ def find_command(cmd, paths = [])
     def main_files_for_component(name)
       Array(dsl.main_files[name])
     end
+
+    def clean_should_skip_component?(name)
+      BowerRails.exclude_from_clean.respond_to?(:include?) &&
```

Comment:
I think we can add default option [here](https://github.com/rharriso/bower-rails/blob/master/lib/bower-rails.rb#L59) and skip this check